app/preferences.html

```diff
@@ -256,6 +256,21 @@
         <input type="radio" value="#567890" name="mainColor" id="morningSwim">
         <label data-i18next="preferences.theme.morningSwim" for="morningSwim"></label>
       </div>
+      <div>
+          <label data-i18next="preferences.theme.opacity" data-same-width-label for="opacity"></label>
+          <input type="range" min="0.1" max="1" step="0.1" list="tickmarksOpacity" name="opacity" data-divisor="1" id="opacity">
+          <datalist id="tickmarksOpacity">
+            <option value="0.2"></option>
+            <option value="0.3"></option>
+            <option value="0.4"></option>
+            <option value="0.5"></option>
+            <option value="0.6"></option>
+            <option value="0.7"></option>
+            <option value="0.8"></option>
+            <option value="0.9"></option>
+          </datalist>
+          <output data-unit="none"></output>
```

Comment:
I didn't specify any unit because I don't know which one I could use, is there even one ?